### PR TITLE
Setup webpack

### PR DESCRIPTION
**Pull request by heatherbooker**
get rid of gulp and browserify
replace with webpack
set up babel-loader, file-loader, sass-loader, html-webpack-plugin
set up webpack-dev-server and hot module reload (instant browser refresh)

### CODE REVIEW

**Comment by heatherbooker** [comment]
@dharness can I get a review

File: build/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (dharness) [inline build/index.html]: @heatherbooker please don't upload the build folder. Add it to .gitignore

File: npm-debug.log [hunk 2]
```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```
Comment (dharness) [inline npm-debug.log]: Please remove this as well

File: react-components.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+MAIN
```
Comment (dharness) [inline react-components.md]: What is the purpose of this file?

**Comment by dharness** [comment]
Reviewed.

**Comment by heatherbooker** [comment]
@dharness made the changes you requested

File: README.md [hunk 4]
```diff
@@ -8,3 +8,16 @@ currently implemented for English learners to learn in French
...
 
 totally a copycat of duolingo
+
+#### to run:  
+
+```
+git clone https://github.com/heatherbooker/frenchly.git  
+cd frenchly  
+npm install  
+npm run start
```
Comment (dharness) [inline README.md:18]: start does not need the 'run' command fyi, though no need to change

**Comment by dharness** [comment]
LGTM

**Pull request merged**